orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/backup_db.sh

```diff
@@ -74,7 +74,7 @@ errorExit() {
   if [[ "$CLOSECONNECTIONS" == "true" ]]; then
     limit_incomming_connection $SERVICE -1
   fi
-  if [ -d "$DATE_DIR" ]; then
+  if [ -d "$DATE_DIR" ] && [[ $DATE_DIR == /var/tmp/* ]]; then
```

Comment:
did you test this ?

Reply:
@votez Yes I tested both backup and restore.